hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1709,6 +1726,13 @@ message SnapshotDiffRequest {
   optional bool forceFullDiff = 7;
 }
 
+message ListSnapshotDiffJobRequest {
```

Comment:
Should it be paginated? It doesn't have to be part of this PR.

Reply:
Other listing commands don't have pagination.

Reply:
okay, in that case we can do it in another PR.

Reply:
Please create a jira for that and resolve this comment.

Reply:
https://issues.apache.org/jira/browse/HDDS-8802